README.md

```diff
@@ -1,127 +1,126 @@
 Docker open-oni
 ===============
 
-Run open-oni (community chronicling america fork) in Docker.
+(Production instructions TBD - for now this is just for doing open-oni dev)
 
-MySQL
------
+open-oni development
+--------------------
 
-```bash
-$ docker run -d \
-$   -p 3306:3306 \
-$   --name mysql \
-$   -e MYSQL_ROOT_PASSWORD=123456 \
-$   -e MYSQL_DATABASE=openoni \
-$   -e MYSQL_USER=openoni \
-$   -e MYSQL_PASSWORD=openoni \
-$   mysql
-```
+First clone open-oni: `git clone git@github.com:open-oni/open-oni.git`.
 
-Now alter the `charset`:
+It's easiest to start with [`./dev.sh`](dev.sh)  - it sets up all the containers in
+order, and makes sure the app is ready to run.  Note for `docker-machine`: use
+`./dev.sh $IP_ADDRESS_OF_MACHINE`.  For Linux users who can't (or don't want
```

Comment:
Okay nevermind, this is obsolete now.  Turns out it was for running mysql commands from your local system, which we've gotten away from since some of us don't have the mysql client installed :)